src/components/Project/ProjectsAPI.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: "https://jsonplaceholder.typicode.com",
+});
+
+export default async function GetProjects() {
+  let errorHandler = true;
+  try {
+    let result = await (await instance.get("/users")).data;
```

Comment:
Why the double await? I think the following would work:
```
const result = await instance.get("/users").data
```

Reply:
@DaleneyRuben I tried it but it doesn't work. Double async await is similar to:
const response = await axios.get("https://jsonplaceholder.typicode.com/users");
const result = await response.data;